mule-extensions-api/api-changes.json

```diff
@@ -29,6 +29,424 @@
           "new": "interface org.mule.runtime.extension.api.runtime.operation.ExecutionContext<M extends org.mule.runtime.api.meta.model.ComponentModel>",
           "annotationType": "org.mule.api.annotation.NoImplement",
           "justification": "This interface is not meant to be implemented by the user"
+        },
+        {
+          "code": "java.annotation.added",
+          "old": "interface org.mule.runtime.extension.api.client.ExtensionsClient",
+          "new": "interface org.mule.runtime.extension.api.client.ExtensionsClient",
+          "annotationType": "org.mule.api.annotation.NoImplement",
+          "annotation": "@org.mule.api.annotation.NoImplement",
+          "package": "org.mule.runtime.extension.api.client",
+          "classSimpleName": "ExtensionsClient",
+          "elementKind": "interface",
+          "justification": "This interface is not meant to be implemented by the user"
+        },
+        {
+          "code": "java.annotation.added",
+          "old": "interface org.mule.runtime.extension.api.client.OperationParameters",
+          "new": "interface org.mule.runtime.extension.api.client.OperationParameters",
+          "annotationType": "org.mule.api.annotation.NoImplement",
+          "annotation": "@org.mule.api.annotation.NoImplement",
```

Comment:
Remove the annotation, package, classSimpleName and elementKind lines, they are redundant